### PR TITLE
Clear changelog

### PR DESCRIPTION
**Pull request by AdRiley**
### Pull Request Description

Reset changelog for 2024.2 release

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

**Review by farmaazon — CHANGES_REQUESTED** [review]
The changelog should be a single file (I put old changelog in a separate due to long time gap and complete rework of GUI). The release script always takes the first section AFAIK.

So, keep the file, just rename "Next Version" to "Enso 2024.2.1" and create new "Next Version" section.

**Comment by AdRiley** [comment]
> The changelog should be a single file (I put old changelog in a separate due to long time gap and complete rework of GUI). The release script always takes the first section AFAIK.
> 
> So, keep the file, just rename "Next Version" to "Enso 2024.2.1" and create new "Next Version" section.

Better?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 #### Enso IDE
 
+#### Enso Enso Standard Library
+
+# Enso 2024.1
```
Comment (farmaazon) [inline CHANGELOG.md]: Now it will be Enso 2024.2

**Review by farmaazon — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AdRiley) [reply CHANGELOG.md]: Fixed

**Pull request merged**